CI_YAML.md

```diff
@@ -168,3 +168,27 @@ Cocoon supports tests that are not owned by Flutter infrastructure. By default,
      scheduler: my_external_location
    ```
 8. Send updates to `https://flutter-dashboard.appspot.com/api/update-task-status` - https://github.com/flutter/cocoon/blob/master/app_dart/lib/src/request_handlers/update_task_status.dart
+
+
+## Scheduling Targets
+
+For targets using the Cocoon scheduler, they can run on:
+ * Presubmit (via GitHub checks)
+ * Postsubmit (via [build dashboard](https://flutter-dashboard.appspot.com/#/build))
+
+By default, all targets should use the Cocoon scheduler.
+
+### Presubmit Features
+
+1. GitHub checks enable targets to run immediately, and are available on the pull request page.
+2. Changes to the ci.yaml will be applied during those presubmit runs.
```

Comment:
This is blocked on https://github.com/flutter/flutter/issues/108645.
Do you mind provide some context in the above bug re why we switched to use TOT commit? Or it was not intended.

Reply:
Sorry, I didn't see the comment. I'll fix the bug (I followed up asking for more details as it seems working to me)